tests/features/call_proposals.feature

```diff
@@ -0,0 +1,164 @@
+    And I fill in "Fax number" with "0488779035" in the "Call for proposals contact" region
+    And I select "Hungary" from "Country" in the "Call for proposals contact" region
+    And I wait for AJAX to finish
+    And I fill in "Street address" with "Back street 3" in the "Call for proposals contact" region
+    And I fill in "Postal code" with "9000" in the "Call for proposals contact" region
+    And I fill in "City" with "Budapest" in the "Call for proposals contact" region
+    And I fill in "Office" with "Call for proposals contact office" in the "Call for proposals contact" region
+    And I fill in "URL" with "mailto:example@email.com" in the "Contact social media links" region
+    And I fill in "Link text" with "Call for proposals contact social link email" in the "Contact social media links" region
+    And I fill in "Media item" with "Contact image" in the "Call for proposals contact" region
+    And I fill in "Caption" with "Call for proposals contact caption" in the "Call for proposals contact" region
+    And I fill in "Press contacts" with "http://example.com/press_contacts" in the "Call for proposals contact" region
+
+    And I fill in "Alternative title" with "Alternative title 1"
+    And I fill in "Navigation title" with "Navi title 1"
+    And I fill in "Redirect link" with "http://example.com"
+
+    And I press "Save"
+    And I should see "My Call for proposals 1 body"
+    And I should see "My Call for proposals 1 reference"
+    And I should see the link "Official Journal publication 1"
+    And I should see "10/25/2020"
+    And I should see the link "http://example.com/2"
+    And I should see "Anti Fraud Information System (AFIS)"
+    And I should see "Audit Board of the European Communities"
+    And I should see "My Document 1"
+
+    And I should see the text "Name of the call for proposals contact"
+    And I should see the text "Call for proposals contact body text"
+    And I should see the text "Call for proposals contact organisation"
+    And I should see the link "http://www.example.com/call_for_proposals_contact"
+    And I should see the text "test@example.com"
+    And I should see the text "0488779033"
+    And I should see the text "0488779034"
+    And I should see the text "0488779035"
+    And I should see the text "Back street 3"
+    And I should see the text "Budapest"
+    And I should see the text "9000"
+    And I should see the text "Hungary"
+    And I should see the link "Call for proposals contact social link email"
+    And I should see the text "Call for proposals contact office"
+    And I should see the link "Contact image"
+    And I should see the text "Call for proposals contact caption"
+    And I should see the link "http://example.com/press_contacts"
+
+    And I should not see "Alternative title 1"
+    And I should not see "Navi title 1"
+
+  @javascript @aabbcc
+  Scenario: Constraints validation of a Call for proposals content through the UI.
```

Comment:
What are we testing here exactly? I can't understand it from this scenario description.

Reply:
I altered the scenario description.